manifests/compose.pp

```diff
@@ -13,9 +13,14 @@
 #   The version of Docker Compose to install.
 #   Defaults to the value set in $docker::params::compose_version
 #
+# [*install_from*]
+#   The URL to use to install docker-compose.
+#   Defaults to the URL recommended by https://docs.docker.com/compose/install/
+#
 class docker::compose(
   $ensure = 'present',
-  $version = $docker::params::compose_version
+  $version = $docker::params::compose_version,
+  $install_from = "https://github.com/docker/compose/releases/download/${version}/docker-compose-${::kernel}-x86_64"
 ) inherits docker::params {
   validate_string($version)
```

Comment:
If you could add a quick `validate_string` for the new `install_url` property as well.